src/main/java/org/stellar/sdk/requests/SSEUtils.java

```diff
@@ -0,0 +1,72 @@
+package org.stellar.sdk.requests;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.internal.sse.RealEventSource;
+import okhttp3.sse.EventSource;
+import okhttp3.sse.EventSourceListener;
+import org.stellar.sdk.responses.GsonSingleton;
+
+import javax.annotation.Nullable;
+import java.net.SocketException;
+
+
+public class SSEUtils {
```

Comment:
This probably shouldn't be `public`. Also, maybe instead of this class we could have a static method (`create`?) that creates a new stream on `SSEManager`?

Reply:
I merged this class into the new SSEStream class and renamed the method.